internal/provider/transfer_endpoint_object_storage.go

```diff
@@ -202,15 +204,15 @@ func transferEndpointObjectStorageSourceSchema() schema.Block {
 					"path_prefix":           schema.StringAttribute{Optional: true},
 					"endpoint":              schema.StringAttribute{Optional: true},
 					"region":                schema.StringAttribute{Optional: true},
-					"use_ssl":               schema.BoolAttribute{Optional: true},
-					"verify_ssl_cert":       schema.BoolAttribute{Optional: true},
+					"use_ssl":               schema.BoolAttribute{Optional: true, Computed: true, Default: booldefault.StaticBool(false)},
```

Comment:
Are you sure about this?

This approach won't work with the most popular case **AWS S3**, because they require SSL by default. They don't have endpoint with plain http as far as I know.

At least default value should be `true`?

Reply:
technically https is not enforced, it has to be done explicitly on buckte policy https://repost.aws/knowledge-center/s3-bucket-policy-for-config-rule. By default both http and https request are allowed

Reply:
It's not a roadblock, I want to be sure that you intentionally marked it as `false`.

Reply:
yeah, left it intentionally to false (basically emulating default value we have in UI)